languages/backupwordpress-en_AU.po

```diff
@@ -0,0 +1,1211 @@
+# Copyright (C) 2015 Human Made Limited
```

Comment:
Not blank, just few lines translated (biweekly is ambiguous and not widely used outside the US). I thought that WP uses the binary, which contains the translated lines sans empties.